### PR TITLE
Update fee page

### LINKED ISSUE #172: Update fees page
### Description

Description and formulas of the L1 data fee need to be updated according to the latest Fjord upgrade on OP Mainnet.

### Location
https://liskhq.github.io/lisk-documentation/fees

### Additional Information
https://docs.optimism.io/stack/transactions/fees#fjord

### PR DESCRIPTION
**Pull request by Tschakki**
### What was the problem?

This PR resolves #172

### CODE REVIEW

File: docs/about-lisk/fees.md [hunk 1]
```diff
@@ -88,23 +93,12 @@ The actual amount of this fee depends on the following input values:
...
-
-`tx_data` is the byte representation of the serialized transaction.
-
-:::note
-`tx_compressed_size` is an estimation of the size that a transaction will occupy in blobs.
-The divisor of 16 represents the storage savings of using blobs vs calldata.
-
-The "L1 Gas used by txn" field of transaction details in [Blockscout](https://blockscout.lisk.com/) contains the `tx_compressed_size` multiplied by 16 (i.e., the `calldata` size).
-:::
+Where `estimatedSize` is an estimation on the size that the transaction will occupy when posted in Ethereum L1 blobs.
```
Comment (gkoumout) [inline docs/about-lisk/fees.md]:
```suggestion
Here `estimatedSize` is an estimation on the size that the transaction will occupy when posted in Ethereum L1 blobs.
```

Comment (gkoumout) [reply docs/about-lisk/fees.md]: Perhaps not so nice to start a sentence with "Where" but not a big deal

**Review by gkoumout — APPROVED** [review]
Nice work! Just left a minor suggestion, otherwise all good!

File: docs/about-lisk/fees.md [hunk 2]
```diff
@@ -44,6 +44,11 @@ At a high level, the L2 fee is the cost to execute your transaction on L2 and th
 transaction_fee = l2_execution_fee + l1_data_fee
 ```
 
+:::note
+Lisks fee calculation is based on the fee system of OP Mainnet, using the latest **Fjord** upgrade.
+Check the [Fees page](https://docs.optimism.io/stack/transactions/fees#fjord) in the Optmism documentation, to learn more about how the fee system works in detail.
```
Comment (TalhaMaliktz) [inline docs/about-lisk/fees.md]:
```suggestion
Lisk's fee calculation is based on the fee system of OP Mainnet, using the latest **Fjord** upgrade.
Check the [Fees page](https://docs.optimism.io/stack/transactions/fees#fjord) in the Optmism documentation, to learn more about how the fee system works in detail.
```

File: docs/about-lisk/fees.md [hunk 3]
```diff
@@ -88,23 +93,12 @@ The actual amount of this fee depends on the following input values:
...
-
-`tx_data` is the byte representation of the serialized transaction.
-
-:::note
-`tx_compressed_size` is an estimation of the size that a transaction will occupy in blobs.
-The divisor of 16 represents the storage savings of using blobs vs calldata.
-
-The "L1 Gas used by txn" field of transaction details in [Blockscout](https://blockscout.lisk.com/) contains the `tx_compressed_size` multiplied by 16 (i.e., the `calldata` size).
-:::
+Here `estimatedSize` is an estimation on the size that the transaction will occupy when posted in Ethereum L1 blobs.
```
Comment (TalhaMaliktz) [inline docs/about-lisk/fees.md]:
```suggestion
Here `estimatedSize` is an estimation of the size that the transaction will occupy when posted in Ethereum L1 blobs.
```

**Review by TalhaMaliktz — APPROVED** [review]
Minor comments! rest LGTM.

**Pull request merged**